assets/dev/js/frontend/handlers/image-carousel.js

```diff
@@ -2,63 +2,132 @@ class ImageCarouselHandler extends elementorModules.frontend.handlers.Base {
 	getDefaultSettings() {
 		return {
 			selectors: {
-				carousel: '.elementor-image-carousel',
+				carousel: '.elementor-image-carousel-wrapper',
+				slideContent: '.swiper-slide',
 			},
 		};
 	}
 
 	getDefaultElements() {
 		const selectors = this.getSettings( 'selectors' );
 
-		return {
+		const elements = {
 			$carousel: this.$element.find( selectors.carousel ),
 		};
+
+		elements.$swiperSlides = elements.$carousel.find( selectors.slideContent );
+
+		return elements;
 	}
 
-	onInit( ...args ) {
-		super.onInit( ...args );
+	getSlidesCount() {
+		return this.elements.$swiperSlides.length;
+	}
 
+	getSwiperSettings() {
 		const elementSettings = this.getElementSettings(),
 			slidesToShow = +elementSettings.slides_to_show || 3,
 			isSingleSlide = 1 === slidesToShow,
 			defaultLGDevicesSlidesCount = isSingleSlide ? 1 : 2,
-			breakpoints = elementorFrontend.config.breakpoints;
-
-		const slickOptions = {
-			slidesToShow: slidesToShow,
-			autoplay: 'yes' === elementSettings.autoplay,
-			autoplaySpeed: elementSettings.autoplay_speed,
-			infinite: 'yes' === elementSettings.infinite,
-			pauseOnHover: 'yes' === elementSettings.pause_on_hover,
+			elementorBreakpoints = elementorFrontend.config.breakpoints;
+
+		const swiperOptions = {
+			slidesPerView: slidesToShow,
+			slidesPerGroup: +elementSettings.slides_to_scroll || 1,
+			loop: 'yes' === elementSettings.infinite,
 			speed: elementSettings.speed,
-			arrows: -1 !== [ 'arrows', 'both' ].indexOf( elementSettings.navigation ),
-			dots: -1 !== [ 'dots', 'both' ].indexOf( elementSettings.navigation ),
-			rtl: 'rtl' === elementSettings.direction,
```

Comment:
Where is this option in the swiper options?

Reply:
Navigation and pagination are handled with <div>'s and classes in Swiper,  it is handled in the PHP.
RTL is handled in the render function.